packages/types/src/app-instance.ts

```diff
@@ -30,12 +30,15 @@ export interface AppIdentity {
   defaultTimeout: number;
 }
 
-export interface AppInterface {
-  addr: string;
+export interface AppFunctionsSigHashes {
```

Comment:
Rename to `AppInterfaceSighashParameters`?

Reply:
Sure